jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -51,6 +51,7 @@ def buildExclusions = [
 
   // PPC BE ------------------------------------------------
   [ /^ppcbe-ubuntu/,                  anyType,     gte(8)  ],
+  [ /^ppcbe-ubuntu/,                  anyType,     gte(12) ],
```

Comment:
```suggestion
  [ /^ppcle-ubuntu/,                  anyType,     gte(12) ],
```

Reply:
I don't understand this change. I'm adding support for ppc64le, why should I change anything about ppcbe or ppcle?

Reply:
Ah, sorry. I see what you mean, I added the wrong exclusion. :eyeroll: I fixed this.